onnxruntime/contrib_ops/cpu/quantization/qlinear_concat.cc

```diff
@@ -98,7 +98,7 @@ Status QLinearConcat::Compute(OpKernelContext* ctx) const {
   std::vector<std::vector<uint8_t>> dynamic_lookup_tables(input_count);
   std::vector<int> dynamic_table_attrs(input_count, 0);
 
-  std::vector<const Tensor*> input_tensors(input_count);
+  InlinedTensorsVector input_tensors(input_count);
```

Comment:
>InlinedTensorsVector [](http://example.com/codeflow?start=2&length=20)

Should we introduce a general purpose InlinedVector typdef with small buffer optimization for general use?
It made sense to introduce a TensorShapeVector since shape is a very widespread use and we do not want to waste much of stack space. However, we are risking template blow out if we do not come up with a general purpose small buffer value.